capstone/src/main/webapp/script.js

```diff
@@ -327,24 +317,30 @@ function isStringEmpty(str) {
 }
 
 /** post request params to send a POST request using fetch() */
-const requestParamPOST = {
-  method: 'POST',
-  headers: {
-    'Content-Type': 'application/json',
-  },
-};
+// const requestParamPOST = {
+//   method: 'POST',
+//   headers: {
+//     'Content-Type': 'application/json',
+//   },
+// };
 
-function getReviews() {
+async function getReviewsEntities(reviews) {
   // TODO(#33): integrate with actual reviews of businesses
-  const review = 'Really good pizza, nice wine, reasonable prices and great music.';
-  getEntities(review);
+  const reviewsEntities = await getEntities(reviews);
+  return reviewsEntities;
 }
 
 /** send POST request to Cloud Natural Language API for entity recognition */
-function getEntities(message) {
-  const url = '/nlp-entity-recognition?message=' + message;
-  fetch(url, requestParamPOST).then((response) => response.json()).then((entities) => {
-    const businessTags = entities;
+function getEntities(messages) {
+  const url = '/nlp-entity-recognition?messages=' + messages;
```

Comment:
Oh I see you put it back in here. But then why don't we just delete the previous instance?

Reply:
oop forget to delete good catch!